tables/no-no-8dot-fallback-6dot-g0.utb

```diff
@@ -1,6 +1,35 @@
-# liblouis: Norwegian 8-dot braille table with fallback to uncontracted 6-dot
+# liblouis: Norwegian literary braille, 8-dot with 6-dot fallback
 #
-#  Copyright (C) 2015 NLB Norwegian library of talking books and braille, http://www.nlb.no/
```

Comment:
I would not change or remove the copyright statements. I think they go together with the license statement and should be there as is. If you want them also as meta data that's fine but the copyright statements should remain as is near the top of the file.

Reply:
OK sure. I'll add those back. Jostein's reasoning was probably that the meta data could replace it.